Tests/js/views/fields/assets/ez-relationlist-editview-tests.js

```diff
@@ -0,0 +1,569 @@
+            );
+
+        },
+
+        "Test validate required": function () {
+            var fieldDefinition = this._getFieldDefinition(true);
+
+            this.view.set('fieldDefinition', fieldDefinition);
+            this.view.set('destinationContents', []);
+            this.view.validate();
+            this.view.render();
+
+            Y.Assert.isFalse(
+                this.view.isValid(),
+                "An empty relation is NOT valid"
+            );
+        },
+
+        "Should render the view when the destinationContent attribute changes": function () {
+            var templateCalled = false,
+                origTpl = this.view.template;
+
+            this.view.template = function (variables) {
+                templateCalled = true;
+                return origTpl.apply(this, arguments);
+            };
+            this.view.set('destinationContents', this.destinationContents);
+            Y.Assert.isTrue(templateCalled, "The template has not been used");
+        },
+
+        "Should render the view when the loadingError attribute changes": function () {
+            var templateCalled = false,
+                origTpl = this.view.template;
+
+            this.view.template = function (variables) {
+                templateCalled = true;
+                return origTpl.apply(this, arguments);
+            };
+            this.view.set('loadingError', true);
+            Y.Assert.isTrue(templateCalled, "The template has not been used");
+        },
+
+
+        "Should fill the destinationContentsIds attribute from the field": function () {
+            Y.Assert.isArray(
+                this.view.get('destinationContentsIds'),
+                "destinationContentsIds should be an array"
+            );
+            for (var i = 0; i <= this.view.get('destinationContentsIds').length; i++) {
+                Y.Assert.areSame(
+                    this.view.get('field').fieldValue.destinationContentIds[i],
+                    this.view.get('destinationContentsIds')[i],
+                    "The destinationContentId of the field value should be the same than the attribute"
+                );
+            }
+        },
+
+        "Should fire the loadFieldRelatedContents event": function () {
+            var loadContentEvent = false,
+                that = this;
+
+            this.view.on('loadFieldRelatedContents', function (e) {
+                Y.Assert.areSame(
+                    that.fieldDefinitionIdentifier,
+                    e.fieldDefinitionIdentifier,
+                    "fieldDefinitionIdentifier is the same than the one in the field"
+                );
+                loadContentEvent = true;
+            });
+            this.view.set('active', true);
+
+            Y.Assert.isTrue(loadContentEvent, "loadContentEvent should be called when changing active value");
+        },
+
+        "Should NOT fire the loadFieldRelatedContents event if field is empty": function () {
```

Comment:
should go in the same test case as "Should fire the loadFieldRelatedContents event"